heron/tools/tracker/src/python/config.py

```diff
@@ -54,7 +57,7 @@ def load_configs(self):
 
     if SPOUT_EXTRA_LINKS_KEY in self.configs:
       for extra_link in self.configs[SPOUT_EXTRA_LINKS_KEY]:
-        self.spout_extra_links[extra_link[SPOUT_TYPE_KEY]] = [self.validate_extra_link(link) for link in extra_link[EXTRA_LINKS_KEY]]
+        self.spout_extra_links[extra_link[SPOUT_TYPE_KEY]] = extra_link[EXTRA_LINKS_KEY]
```

Comment:
why not need to validate?

Reply:
The way we were doing validation worked for extra links with only pre-defined params, but it's hard to pre-define all possible params beforehand. In addition, different spout types could have different sets of params.

Reply:
kk. thx